extensions/cornerstone/src/services/SyncGroupService/SyncGroupService.ts

```diff
@@ -152,6 +154,8 @@ export default class SyncGroupService {
         return;
       }
 
+      this.unRegisterSpatialRegistration(synchronizer);
```

Comment:
can we filter and only do this for sync synchronizer?

Reply:
Thanks for the review, @sedghi; I sure can do it;  Must does not seem to exists a way to differentiate the synchronizer other than the name.

Is checking the name of the sincronizer enough for this situation?

Reply:
sure

Reply:
Thanks, @sedghi. I have updated the code, and it is working as expected now. Let me know if you think I should improve anything else.